src/runtime/rpc/rpc_session.cc

```diff
@@ -1190,21 +1192,39 @@ void RPCSession::EventHandler::HandlePackedCall() {
   CHECK_EQ(state_, kRecvCode);
 }
 
-PackedFunc WrapTimeEvaluator(PackedFunc pf, TVMContext ctx, int number, int repeat) {
-  auto ftimer = [pf, ctx, number, repeat](TVMArgs args, TVMRetValue *rv) {
+PackedFunc WrapTimeEvaluator(PackedFunc pf,
+                             TVMContext ctx,
+                             int number,
+                             int repeat,
+                             int min_repeat_ms) {
+  auto ftimer = [pf, ctx, &number, repeat, min_repeat_ms](TVMArgs args, TVMRetValue *rv) {
     TVMRetValue temp;
     std::ostringstream os;
     // skip first time call, to activate lazy compilation components.
     pf.CallPacked(args, &temp);
```

Comment:
I wonder if this definition (1 + number * repeat) is the correct formulation after we have introduced `min_repeat_ms`. The goal is to start measurement in the correct power state, which we will likely do if we bump up `number` over and over again for the same `time_evaluator` call. However, let's say that `number` is now sufficient and we get to a fresh `time_evaluator` call. In this case I am not sure `1+` will be enough to get the hardware into the right state if necessary. Should we consider `number*(1+repeat)`?

Reply:
Yes, the definition is not correct. I will add a note to the doc string of `min_repeat_ms` but keep this definition here.